plugin/user_plugin_server.py

```
@@ -0,0 +1,559 @@
+from fastapi import FastAPI, HTTPException, Request, Query
+from fastapi.responses import JSONResponse
+from typing import Optional
+import asyncio
+import threading
+import logging
+from datetime import datetime
+from typing import Any, Dict, List, Optional
```

Ruff (0.14.7)

Redefinition of unused `Optional` from line 3

Remove definition: `Optional`

(F811)